DependencyInjection/Configuration.php

```diff
@@ -408,7 +410,7 @@ public function getConfigTreeBuilder()
                             ->booleanNode('use_locking')->defaultFalse()->end() // stream and rotating
                             ->scalarNode('filename_format')->defaultValue('{filename}-{date}')->end() //rotating
                             ->scalarNode('date_format')->defaultValue('Y-m-d')->end() //rotating
-                            ->scalarNode('ident')->defaultFalse()->end() // syslog and syslogudp
+                            ->scalarNode('ident')->defaultValue('php')->end() // syslog and syslogudp
```

Comment:
This looks like a bc break. The https://github.com/Seldaek/monolog/blob/main/src/Monolog/Handler/SyslogHandler.php does not have 'php' as default value. So it used an empty string (casted from false) before.

Reply:
But we talking about SyslogUdpHandler https://github.com/Seldaek/monolog/blob/main/src/Monolog/Handler/SyslogUdpHandler.php#L49

Reply:
yes, but @Tobion is right here, this configuration is also used by both `Syslog` and `SyslogUdp`.
The change should keep compatibility with the 2 handlers.

Could you also please add a test to cover this case?